### PR TITLE
Supabase Streaming Queries

### PR DESCRIPTION
**Pull request by pinkeshmars**
### Description
Supabase Streaming Queries

[Linear ticket](https://linear.app/flutterflow/issue/DEVR-662/documentation-for-supabase-streaming-queries) and [magic word](https://linear.app/docs/github#link-prs) Fixes DEVR-662 

## Type of change
- [ ] Typo fix 
- [x] New feature 
- [x] Enhancement to current docs
- [ ] Removed outdated references
- [x] Update assets

### CODE REVIEW

File: docs/resources/control-flow/backend-logic/backend-query/query-collection.md [hunk 1]
```diff
@@ -113,4 +149,21 @@ You can follow similar steps for using the record data on the other widgets as w
...
+</div>
+<p></p>
+</TabItem>
+</Tabs>
+
+## FAQs
+<details>
+<summary>Why are real-time updates for Supabase not working?</summary>
+<p>
+First, ensure that the **Single Time Query** option is disabled in the query where you've added it. Then, verify that the real-time feature is enabled for your table in Supabase project. You can find this option in the top-right corner of the table viewer.
```
Comment (leighajarett) [inline docs/resources/control-flow/backend-logic/backend-query/query-collection.md:161]: "for your table in your Supabase project"

**Review by leighajarett — APPROVED** [review]
Small nit but otherwise good to go

**Comment by leighajarett** [comment]
Actually I was just looking at existing documentation and I'm a bit confused, we have a section here: https://docs.flutterflow.io/integrations/database/supabase/database-actions/#query-rows-action

It's confusing to me that both of these sections exist? We should probably consolidate these and just link to one another, unless I am missing something

File: docs/resources/control-flow/backend-logic/backend-query/query-collection.md (hunk 1 above)
Comment (pinkeshmars) [reply docs/resources/control-flow/backend-logic/backend-query/query-collection.md:161]: ok, corrected question text as 'Why aren't real-time updates working for my table in Supabase project?'

**Comment by pinkeshmars** [comment]
> Actually I was just looking at existing documentation and I'm a bit confused, we have a section here: https://docs.flutterflow.io/integrations/database/supabase/database-actions/#query-rows-action
> 
> It's confusing to me that both of these sections exist? We should probably consolidate these and just link to one another, unless I am missing something

@leighajarett The section you've mentioned (https://docs.flutterflow.io/integrations/database/supabase/database-actions/#query-rows-action) specifically focuses on how to add queries using actions, where the streaming option (Single Time Query) isn't available.

The section we updated in this PR, on the other hand, describes how to add queries directly on a page or widget, so that they automatically fire whenever a user navigates to the page containing the query.

Both sections serve different use cases.

**Comment by PoojaB26** [comment]
@leighajarett As Pinkesh mentioned, the difference seems to be Backend Query vs Actions. This change is on Backend Query if I understand correctly.

**Comment by leighajarett** [comment]
I think we should still need to link the queries from the Supabase and Firebase integrations page... since theyre part of the integration and critical to using it.

**Pull request merged**